modules/apps/petra/petra-json-web-service-client/src/test/java/com/liferay/petra/json/web/service/client/internal/JSONWebServiceClientImplGetTest.java

```diff
@@ -79,8 +85,17 @@ public void testCommunicationFailureOnGetWithRetryable() throws Exception {
 
 		jsonWebServiceClientImpl.activate(properties);
 
-		jsonWebServiceClientImpl.doGet(
-			"/testGet/", Collections.<NameValuePair>emptyList());
+		try {
+			jsonWebServiceClientImpl.doGet(
+				"/testGet/", Collections.<NameValuePair>emptyList());
+
+			Assert.fail();
+		}
+		catch (JSONWebServiceTransportException jsonwste) {
```

Comment:
@dantewang @yuhai Please note that I did not include this commit.
Reason is the assert logic is actually not necessary, it will always pass, cause the catch block already checked the class type, we need more detailed assert logic here.

Tina.